pkg/backend/diy/backend.go

```diff
@@ -795,29 +799,66 @@ func (b *diyBackend) ListStacks(
 		return nil, nil, err
 	}
 
+	// Get the max parallel value from environment variable or use a default value
+	maxParallel := b.Env.GetInt(env.DIYBackendParallel)
```

Comment:
```suggestion
	parallel := b.Env.GetInt(env.DIYBackendParallel)
```
Just for consistency